app/src/main/java/fr/neamar/kiss/forwarder/ExperienceTweaks.java

```diff
@@ -210,6 +213,16 @@ private void doAction(String action) {
                             mainActivity.hideKeyboard();
                         }
                         break;
+                    case "launch-pojo": {
+                        String launchId = prefs.getString(source + "-launch-id", "");
+                        Pojo item = KissApplication.getApplication(mainActivity).getDataHandler().getItemById(launchId);
+                        if (item != null) {
+                            // don't send null parent if (item instanceof ContactsPojo)
```

Comment:
Why ContactPojo here?

Reply:
If we add contacts as a launch option and leave `Result.fromPojo(null, item);` then the result might crash as it's the only one that uses that parameter